Appraisals

```diff
@@ -1,4 +1,4 @@
-%w[4.2 5.1 5.2 6.0].each do |version|
+%w[4.2 5.1 5.2 6.0 6.1].each do |version|
```

Comment:
it looks like you haven't run `appraisals generate`. Can you please add all the version defined here to the CI and regenerate all the gemfiles?

Reply:
I'm not sure about the meaning of the question. The gemfile for 6.1.0 is in the expected folder.

Reply:
I see what you mean @pyromaniac. Git is wrongly assuming there is a rename+change from the 4.2 gemfile to the 6.1 gemfile. But it really is a completely new file, the gemfiles for 4.2 to 5.2 were removed to be able to merge into toptal/active_data fork that relies on a different CI. I'll restore the previous changes so it will be clearer.